src/Symmetric.jl

```diff
@@ -0,0 +1,217 @@
+@doc doc"""
+    SymmetricMatrices{n,T} <: Manifold
+
+The [`Manifold`](@ref) $ \operatorname{Sym} (n)$ consisting of the `T`-valued, symmetric matrices of size $ n\times n$, i.e. the set 
+
+$\operatorname{Sym}(n) = \bigl\{A \in \mathbb F^{n\times n} \big| A^{\mathrm{T}} = A \bigr\}$
+
+where $\mathbb F \in \{ \mathbb R, \mathbb C\}$.
+For `T`, there is a distinction between 
+* `T = Real`: real-valued matrix entries
+* `T = Complex`: complex-valued matrix entries
+
+Being slightly inefficient, the matrices are safed as $n\times n$ arrays despite the redundance of the lower triangle.
+
+
+# Constructor
+
+    SymmetricMatrices(n::Int,T::Type = Real)
+
+"""
+struct SymmetricMatrices{N,T} <: Manifold end
+SymmetricMatrices(N::Int,T::Type = Real) = SymmetricMatrices{N,T}()
+
+function representation_size(::SymmetricMatrices{N,T}) where {N,T}
```

Comment:
No need for `T` here

Reply:
How can I leave out T here? Because the following code results in an (UndefVar) error: 
`function representation_size(::SymmetricMatrices{N,T}) where {N}
    return (N,N)
end`

Reply:
Yes, then `T` is unknown, just really leave it out completely writing line 24 as
````Julia
function representation_size(::SymmetricMatrices{N}) where {N}
````